### PR TITLE
COMMERCE-11160 OOTB Account Role of Supplier to manage Channel

### PR DESCRIPTION
**Pull request by dsitu**
https://liferay.atlassian.net/browse/COMMERCE-11160

### CODE REVIEW

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:stop

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
@dsitu , the PR needs to be updated if you are trying to use COMMERCE-11163 commits.

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceAccountRoleHelperImpl.java [hunk 1]
```diff
@@ -273,6 +292,16 @@ else if (name.equals(AccountRoleConstants.ROLE_NAME_SUPPLIER)) {
...
 
+			companyResourceActionIds.put(
+				"com.liferay.commerce.tax",
+				new String[] {
+					CPActionKeys.VIEW_COMMERCE_PRODUCT_TAX_CATEGORIES
+				});
+
+			companyResourceActionIds.put(
+				Group.class.getName(),
+				new String[] {ActionKeys.ADD_LAYOUT, ActionKeys.UPDATE});
```
[the remark this answers is not in the document]
Comment (dsitu) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceAccountRoleHelperImpl.java:303]: This is needed for Health Check, Category Display Pages, and Product Display Pages configurations.

[the remark this answers is not in the document]
Comment (luca-pellizzon) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceAccountRoleHelperImpl.java:303]: I don't think those things need to be accessible by the Supplier tho

[the remark this answers is not in the document]
Comment (dsitu) [reply modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/util/CommerceAccountRoleHelperImpl.java:303]: ahh kk, I will remove them for the Supplier Role, and document them on the ticket

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
ci:test:commerce

**Comment by dsitu** [comment]
ci:test:relevant

**Review by luca-pellizzon — APPROVED** [review]

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
Looks like I caused a regression with my changes. This test is failing CanNotEditChannelCurrencyAsUserWithoutEditPermission.

**Comment by 22LucaGiannotta22** [comment]
@luca-pellizzon yes, I think the test needs to be updated based on what you just said

**Comment by dsitu** [comment]
ci:test:sf

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by dsitu** [comment]
@luca-pellizzon @22LucaGiannotta22 I added a commit with some fixes for the test https://github.com/liferay-commerce/liferay-portal/pull/3917/commits/86f2a7fd1369fe3cc0d36c158a58e29e19dbdc62.

**Comment by dsitu** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:test:commerce-functional

**Comment by 22LucaGiannotta22** [comment]
ci:test:relevant

**Comment by 22LucaGiannotta22** [comment]
ci:forward

**Pull request closed without merging**